.circleci/config.yml

```diff
@@ -50,3 +60,28 @@ workflows:
             - deploy_production:
                   requires:
                       - approve_to_deploy_in_production
+
+    pull_up:
+        jobs:
+            - pull-up/pull-up:
+                  from: "5.0"
+                  into: "master"
+                  branch_prefix: "50_to_master"
+                  github_token: MICHEL_TAG_TOKEN
+                  github_username: "micheltag"
+                  slack_webhook: "${SLACK_URL_PULL_UP}"
+                  filters:
+                      branches:
+                          only:
+                              - "5.0"
```

Comment:
are you sure this WF  has to be reported to master?

Reply:
It has a filter on 5.0 so I guess it's harmless on master?

But yes, maybe it doesn't make sense.